<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Composer</class>
 <widget class="QWidget" name="Composer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="composerGraphs">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_9">
      <layout class="QVBoxLayout" name="sinusoidals_area">
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_10">
      <layout class="QVBoxLayout" name="composing_area">
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="composerControls">
     <item>
      <widget class="QGroupBox" name="componentsBox">
       <property name="title">
        <string>Components</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="deleteComponentBtn">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 105, 107);</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="componentList"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="amplitude">
          <item>
           <widget class="QLabel" name="amplitude_label">
            <property name="text">
             <string>Amplitude</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="amplitude_input">
            <property name="minimum">
             <double>-9000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="phase">
          <item>
           <widget class="QLabel" name="phase_label">
            <property name="text">
             <string>Phase</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="phase_input">
            <property name="minimum">
             <double>-9000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="frequency">
          <item>
           <widget class="QLabel" name="frequency_label">
            <property name="text">
             <string>Frequency</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="frequency_input">
            <property name="minimum">
             <double>-9000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addComponentBtn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="examplesBox">
       <property name="title">
        <string>Examples</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QComboBox" name="examplesMenu">
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveExampleBtn">
          <property name="text">
           <string>Save Example</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startSamplingBtn">
          <property name="text">
           <string>Start Sampling</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
